clap-v3-utils/src/keypair.rs

```diff
@@ -1000,6 +999,15 @@ pub fn keypair_from_path(
     keypair_name: &str,
     confirm_pubkey: bool,
 ) -> Result<Keypair, Box<dyn error::Error>> {
+    encodable_key_from_path(matches, path, keypair_name, confirm_pubkey)
+}
+
+fn encodable_key_from_path<K: EncodableKey>(
```

Comment:
Will ElGamal keypairs and authenticated encryption keys support all the same Sources that Keypair does?

Reply:
Yeah, I am not sure how many users will actually use some of these sources to derive ElGamal and AE keys, but they should support all sources except for `Usb`. I think this should be fine since `keypair_from_path` (or now `encodable_key_from_path`) errors on `Usb` anyway.